portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,8 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
```

Comment:
This was part of your other PR (pull request) right?

You want to rebase your current set of changes onto master branch first and then push up your rebased changes to your current PR.